internal/core/metadata/uom/bootstrap.go

```diff
@@ -36,14 +38,21 @@ func BootstrapHandler(_ context.Context, _ *sync.WaitGroup, _ startup.Timer, dic
 		return true
 	}
 
-	contents, err := os.ReadFile(filepath)
+	requestTimeout, err := time.ParseDuration(config.Service.RequestTimeout)
+	if err != nil {
+		lc.Warnf("Failed to parse Service.RequestTimeout configuration value: %v, using default value", err)
+		requestTimeout = 15 * time.Second
+	}
+
+	secretProvider := bootstrapContainer.SecretProviderFrom(dic.Get)
+	contents, err := file.Load(filepath, requestTimeout, secretProvider)
```

Comment:
Plan from our slack discussion is to refactor to handle getting/using the timeout internally, i.e. remove the requestTimeout parameter.